config.h

```diff
@@ -160,10 +160,12 @@
  * Pin assignments
  */
 #define SPI_BUS         HSPI
-#define RESET_PIN       33
-#define DC_PIN          23
-#define CS_PIN          15
-#define BUSY_PIN        27
+#define DIN_PIN         -1 // COPI
+#define CLK_PIN         -1 // SCK
+#define CS_PIN          15 // CS
+#define DC_PIN          23 // Any OUTPUT pin
+#define RESET_PIN       33 // Any OUTPUT pin
+#define BUSY_PIN        27 // Any INPUT pin
```

Comment:
> Instead of 'arbitrary', I'd mention that DC and RESET must be output capable pins since pins 34, 35, 36, and 39 are input only. I actually ran into that when I first built this by wiring the DC to pin 35.

This is a good point - I updated the comment to note the type of pin needed (e.g. input/output)